MainWindow.xaml.cs

```diff
@@ -1,58 +1,251 @@
 using System.Collections.Generic;
+using System.Text.RegularExpressions;
 using System.Windows;
 using System.Windows.Controls;
 using System.Windows.Input;
 
+using ZC.TimeCalculator.Resources.Extension;
```

Comment:
Don't forget to run a code cleanup before each commit to avoid this kind of excess space

Reply:
It was on purpose to separate usings of System and ZeaCold but I can unite them if it's better